src/bootstrap_gui.cpp

```diff
@@ -220,7 +220,7 @@ bool HandleBootstrap()
 	if (BlitterFactory::GetCurrentBlitter()->GetScreenDepth() == 0) goto failure;
 
 	/* If there is no network or no freetype, then there is nothing we can do. Go straight to failure. */
```

Comment:
The comment may need an update ;)

Reply:
Initially, I was thinking the same. But look a line lower. It is still correct :)